packages/fluentui/e2e/tests/treeKeyboardNavigation-test.ts

```diff
@@ -46,4 +46,21 @@ describe('Tree keyboard navigation', () => {
     await e2e.expectCount(selectors.treeItem, 3);
     await e2e.isFocused(selectors.treeItemAt(1));
   });
+
+  it('Should set focus based on first letter', async () => {
```

Comment:
might be worth using [it.each](https://jestjs.io/docs/en/api#testeachtablename-fn-timeout)

Reply:
for me I wanted to test for 3 cases:
- press letter key -> found item that follows the current focused item -> focus on it
- press letter key -> found item above the current focused item -> focus on it
- press letter key -> didn't find any thing -> focus stays.

So I think `it.each` is a not needed. But good to know ❤️, I might use it for the follow up task on virtualized tree :)